packages/components/src/components/Trigger/Trigger.js

```diff
@@ -0,0 +1,154 @@
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import LinkIcon from '@carbon/icons-react/lib/launch/16';
+import { injectIntl } from 'react-intl';
+import React from 'react';
+import { urls } from '@tektoncd/dashboard-utils';
+import { Link } from 'react-router-dom';
+import './Trigger.scss';
+import { Table } from '@tektoncd/dashboard-components';
+
+const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
+  const tableHeaders = [
+    {
+      key: 'name',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.name',
+        defaultMessage: 'Name'
+      })
+    },
+    {
+      key: 'value',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.value',
+        defaultMessage: 'Value'
+      })
+    }
+  ];
+
+  let triggerParams = [];
+
+  if (trigger.params) {
+    triggerParams = trigger.params.map(param => ({
+      id: param.name,
+      name: param.name,
+      value: param.value
+    }));
+  }
+
+  let interceptorValues = [];
+
+  if (trigger.interceptor) {
+    if (trigger.interceptor.header) {
+      interceptorValues = trigger.interceptor.header.map(header => ({
+        id: header.name,
+        name: header.name,
+        value: header.value
+      }));
+    }
+  }
+
+  return (
+    <>
+      <div className="triggerresourcelinks">
+        <span className="resourcekind">TriggerBinding</span>
+        <Link
+          to={urls.triggerBindings.byName({
+            namespace: eventListenerNamespace,
+            triggerBindingName: trigger.binding.name
+          })}
+        >
+          <span className="linkicon">
+            <LinkIcon />
+          </span>
+          <span className="truncate-text-end" title={trigger.binding.name}>
+            {trigger.binding.name}
+          </span>
+        </Link>
+      </div>
+      <div className="triggerresourcelinks">
+        <span className="resourcekind">TriggerTemplate</span>
+
+        <Link
+          to={urls.triggerTemplates.byName({
+            namespace: eventListenerNamespace,
+            triggerTemplateName: trigger.template.name
+          })}
+        >
+          <span className="linkicon">
+            <LinkIcon />
+          </span>
+          <span className="truncate-text-end" title={trigger.template.name}>
+            {trigger.template.name}
+          </span>
+        </Link>
+      </div>
+
+      <div className="triggerInfo">
+        <h2>
+          {intl.formatMessage({
+            id: 'dashboard.triggerDetails.triggerParameters',
+            defaultMessage: 'Parameters'
+          })}
+        </h2>
+
+        <Table // If anyone knows how to make this not go all grey on-hover please fix
```

Comment:
Looks like Carbon doesn't want to support or encourage use of nested DataTables: https://github.com/carbon-design-system/carbon-components-react/pull/2108#issuecomment-477664867
However, you may find this useful: https://github.com/carbon-design-system/carbon/pull/2222 (for Carbon 9 but might still be applicable)